src/code.cloudfoundry.org/vxlan-policy-agent/converger/converger.go

```diff
@@ -155,12 +155,14 @@ func (m *SinglePollCycle) SyncASGsForContainers(containers ...string) error {
 				})
 				chain, err := m.enforcer.EnforceRulesAndChain(ruleset)
 				if err != nil {
-					errors = multierror.Append(errors, fmt.Errorf("enforce-asg: %s", err))
+					if _, ok := err.(*enforcer.CleanupErr); ok {
+						m.logger.Error("failed-to-cleanup", err)
```

Comment:
Why don't we use `multierror.Append(errors, fmt.Errorf("failed-to-cleanup %s", err))` here?